README.md

```diff
@@ -13,3 +13,35 @@ Follow these steps to set up the project:
 To start the program: `./mvnw spring-boot:run`
 
 If you are using IntellJ IDEA, please install this plugin that provides formatter support: https://github.com/spring-io/spring-javaformat#intellij-idea
+
+## Java Project Custom Data Structure
+Use JavaParser and convert the outputted AST into the custom JavaProject class described below.
+
+### JavaProject Class
+Contains a mapping of package name to class names   
+Contains a mapping of class names to JavaClass classes  
```

Comment:
I think it would look better if the keywords, such as JavaClass, are highlighted as `JavaClass`.